tests/istio_testcases/example_policies/istio-egress-test-partial-flow1/istio-egress-test-partial-flow1-scheme.yaml

```diff
@@ -0,0 +1,18 @@
+namespaceList: ./
+podList: ./
+
+networkConfigList:
+  - name: istio-egress
+    resourceList:
+      - ./resources
+    expectedWarnings: 3
```

Comment:
for this test I can see two related warnings:
```
Warning: Ignoring partially specified connections through egress gateway: mesh-to-egress-gateway connections are missing, while egress-gateway-to-external-service connections are specified in virtual service default/httpbin, route number 1, gateway istio-system/my-gateway.
Warning: no valid VirtualServices found. Ignoring istio ingress/egress gateway traffic
```

Can we avoid the second warning here?
Also, in cases the second warning is relevant, can it specify ingress or egress only?

Reply:
Avoided printing the second warning in the above case.
Removed 'ingress/egress' from the second warning, apparently for invalid/partially defined virtual services / gateways we cannot always detect whether it is an ingress or egress flow.